app/assets/javascripts/models/scorecard.js

```diff
@@ -6,6 +6,7 @@ App.Scorecard = DS.Model.extend({
   user: DS.belongsTo('user', { async: true }),
   userInitials: DS.attr('string'),
   turns: DS.hasMany('turn', { async: true }),
+  isStarted: DS.attr('boolean'),
```

Comment:
Ah dang, me and Kelly were talking about whether it would have to be computed or not and I think we never followed up on it. Sort of a bummer to have to duplicate that logic on the frontend and the backend, but it doesn't seem like there's an alternative.